docs/index.md

```diff
@@ -1,13 +1,58 @@
-# Overview
+# Radiotherapy AI's QMS and TechDoc
+
+Radiotherapy AI's Quality Management System and Technical Documentation. Built
+on top of [GitHub](https://github.com/RadiotherapyAI/RadiotherapyAI) and `git`,
+[OpenRegulatory templates](https://github.com/openregulatory/templates), and
+[Jupyter Book](https://jupyterbook.org/).
+
+All source for the Medical Device, the Quality Management System, and the
+Technical Documentation are available at
+<https://github.com/RadiotherapyAI/RadiotherapyAI>.
+
+## Disclaimer while QMS and TechDocs are under initial development
 
 This Quality Management System and this Technical Documentation are currently
 under active development and there is likely to be both mistakes and omissions.
 The creation of these documents is being shared publicly out in the open in the
 hope that the development of these documents is able to help other SaMD
 companies create and regulate their products and help their patients.
 
-To see the draft introduction document, see
-{doc}`drafts/introduction`.
+## Device
+
+These documents corresponds to version {{device_version}} of the
+{{device_name}} software. Currently this software is pre-release, and both the
+software and this document is expected to change before it is available for
+clinical use. Please reference the version of these documents that is supplied
+to you with the software itself.
+
+### Product Details
+
+- **Manufacturer Name**: {{device_manufacturer}}
+- **Manufacturer's Address**: 17 Grampian Place Tatton NSW 2650 Australia
+- **Software as a Medical Device Identification**: {{device_name}} {{device_version}}
+
+### Overview
+
+Radiotherapy AI's contour recommendations product is deployed onsite as a DICOM
+server. It is designed to sit between your centre's CT scanner and your
+treatment planning system.
+
+When patients undergo their simulation images are auto-sent through to
```

Comment:
simulation, images

Reply:
Thanks @sjswerdloff :), I've pulled this out into its own issue to address when I touch that document next :slightly_smiling_face:.